locales/en/plugin__kubevirt-plugin.json

```diff
@@ -52,6 +52,7 @@
   "Access to public templates": "Access to public templates",
   "Actions": "Actions",
   "Active descheduler": "Active descheduler",
```

Comment:
it looks like we still have at least on appearence of "Active descheduler" instead of "Active Descheduler"
please try to find it and change it as well